cdk/Makefile

```diff
@@ -8,10 +8,10 @@ cdksynth: buildcdk
 	npx cdk -v synth ClipsUI
 
 deploy: build
-	npx cdk -v deploy ClipsBackend --outputs-file tempvars.json
+	cdk deploy -v --outputs-file tempvars.json ClipsBackend
 	node repack.js
 	rm tempvars.json
-	npx cdk -v deploy ClipsUI
+	cdk deploy -v ClipsUI
```

Comment:
To remain consistent with our other IVS demos, please run the CDK deploy scripts using `npx`.

Reply:
scripts homologated with npx ;)